tcms/testruns/templates/testruns/get.html

```diff
@@ -0,0 +1,102 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+
+{% block title %}TR-{{ test_run.pk }}: {{ test_run.summary }}{% endblock %}
+{% block body_class %}cards-pf{% endblock %}
+
+{% block contents %}
+
+<div class="container-cards-pf">
+    <!-- Important:  if you need to nest additional .row within a .row.row-cards-pf, do *not* use .row-cards-pf on the nested .row  -->
+    <h1 class="col-md-12" style="margin-top: 0">
+        <span id="test_case_pk"
+            data-pk="{{ object.pk }}"
+            data-perm-remove-tag="{{ perms.testcases.delete_testcasetag }}"
+            data-perm-remove-component="{{ perms.testcases.delete_testcasecomponent }}"
+            data-perm-remove-plan="{{ perms.testcases.delete_testcaseplan }}"
+            data-perm-remove-bug="{{ perms.testcases.delete_bug }}"
```

Comment:
These permissions don't look right

Reply:
this is copy pasted from the test case template, but I think I can even remove these tag, because I don't see why we need them